### ISSUE TITLE
Local Development Doc Update : backup

### ISSUE THREAD

**Issue by kyletaylored** [pantheon-systems/documentation#2117]
Re: [Local Development](https://pantheon.io/docs/local-development)
## Issue Description:
`'backup:get' is not a registered command.`
**terminus backup** is no longer a valid command (currently running 1.0.0-beta.2).

## Suggested Resolution 
I didn't see any backup options under the **site** command, so I'm not sure if it's been moved, or removed completely.

**Comment by rachelwhitton** [comment]
Thanks for opening this issue @kyletaylored! The `backup:get` command _should_ have been available in 1.0.0-beta.2 
```
$ terminus self:info
 ----------- ------------------------------------------
  PHP         /usr/local/Cellar/php70/7.0.14_7/bin/php
  binary
  PHP         7.0.14
  version
  php.ini     /usr/local/etc/php/7.0/php.ini
  used
  Terminus
  project
  config
  Terminus    /Users/rachel/vendor/pantheon-systems/te
  root dir    rminus
  Terminus    1.0.0-beta.2
  version
  Operating   Darwin Kernel Version 16.4.0: Thu Dec 22
  system      22:53:21 PST 2016;
              root:xnu-3789.41.3~3/RELEASE_X86_64
 ----------- ------------------------------------------
```
```
$ terminus backup:get -h
Usage:
  backup:get [options] [--] <site_env>
  backup:get terminus backup:get <site>.<env>
  backup:get terminus backup:get awesome-site.dev --file=2016-08-18T23-16-20_UTC_code.tar.gz
  backup:get terminus backup:get awesome-site.dev --element=code

Arguments:
  site_env                 Site & environment in the format `site-name.env`

Options:
      --file[=FILE]        [filename.tgz] Name of backup file
      --element[=ELEMENT]  [code|files|database|db] Backup element to retrieve
  -h, --help               Display this help message
  -q, --quiet              Do not output any message
  -V, --version            Display this application version
      --ansi               Force ANSI output
      --no-ansi            Disable ANSI output
  -n, --no-interaction     Do not ask any interactive question
  -y, --yes                Answer all confirmations with "yes"
  -v|vv|vvv, --verbose     Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Help:
  Displays the download URL for a specific backup or latest backup.
```

You can upgrade to the latest version [using the Terminus installer](https://pantheon.io/docs/terminus/install).

**Comment by kyletaylored** [comment]
Ah yea, I should've updated this, but I realized I had conflicting versions of terminus running at the same time.

Thanks!
